test/e2e/startup_spec.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  'Startup Test': (browser) => {
+    browser
+      .url('http://localhost:4000')
```

Comment:
I think you can put the default URL in the config....which is useful when you want to move this saucelabs for example, or test against the live site etc.